### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Response Compression_Screen Reader: Narrator is not reading the role of “Storage Quota” & “Min File Size” as “Spin Box”

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#125]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0
Screen Reader: Narrator
**WCAG ID Violated:** WCAG1.3.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Edge; #Narrator; #WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Response Compression and press enter
6.	Start Screen Reader (Narrator).
7.	Validate that screen reader is reading the role of all the controls.

**Actual Result:**
Narrator is not reading the role of “Storage Quota” and “Min File Size” as “Spin Box”. Also Narrator is reading as Minimum value as 0 and maximum value as 0.

**Expected Result:**
Narrator should read the role of “Storage Quota” and “Min File Size” as “Spin Box”

**User Impact:**
Users with visual disabilities may not be aware of the functionality of the control if screen reader is not reading its role

**Attaching video for your reference:**
[Bug 125_Narrator_SpinBox_Role.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327150/Bug.125_Narrator_SpinBox_Role.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
